packages/react-data-grid-react-window-grid/src/components/DataGrid/DataGrid.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { useDataGrid_unstable } from './useDataGrid';
+import {
+  renderDataGrid_unstable,
+  useDataGridStyles_unstable,
+  useDataGridContextValues_unstable,
+  DataGridProps,
+} from '@fluentui/react-components';
+import type { ForwardRefComponent } from '@fluentui/react-utilities';
+
+export const DataGrid: ForwardRefComponent<DataGridProps> = React.forwardRef(
+  (props, ref) => {
```

Comment:
```suggestion
export const DataGrid = React.forwardRef<HTMLDivElement, DataGridProps>(
  (props, ref) => {
```

there are some reasons internally in Fluent UI why we have a helper type for forwardRef, but they shouldn't apply here IMO

Reply:
Got it thanks